src/media/js/addon/components/addon.js

```diff
@@ -71,6 +72,13 @@ export class Addon extends React.Component {
           </dd>
         </dl>
 
+        {this.props.description &&
+          <di>
```

Comment:
It's how I'm fixing the stupid JSX "Adjacent JSX elements must be wrapped" rule for now. Since this will never be how it looks publicly, it doesn't matter that it's not a real element.